.github/workflows/format.yml

```diff
@@ -0,0 +1,26 @@
+name: Format Project
```

Comment:
Do we want these as separate actions or a single ci/cd one? We've generally had a single one, but I think splitting them is a good option. I imagine this'll make the output clearer in each PR

Reply:
Yeah I appreciate the more granular output. I could move it into one job like this to get more re-use out of `build` / `install` + get the granular output. Did this [here](https://github.com/interunit/ui/blob/release/v0.1.0/.github/workflows/pr-checks.yml) previously. 

Will follow up with a PR addressing this feedback once I have a clearer idea of what I want to be running in Actions.